### PR TITLE
0.15.0

### LINKED ISSUE #224: Please cut new 'Beta' release
- [x] quality checklist complete (sduskis)
- [x] README updated (sduskis)
- [x] cut release (stephenplusplus)
- [x] CI completes / publish package to npm (auto)
- [x] refresh API reference docs on cloud.google.com (danoscarmike)

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #224

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
Sorry for the conflicts @stephenplusplus that was me.  All of them landed.

**Comment by stephenplusplus** [comment]
Tentative notes (please review, @sduskis):

---

This library is now considered to be in **beta**. This means it is expected to be mostly stable while we work toward a general availability release; however, complete stability is not guaranteed. We will address issues and requests against beta libraries with a high priority.

## Breaking Changes

- (#160): Node v4 support dropped.

## Features

- (#161): Allow raw table mutations.
- (#182): Add `table.waitForReplication()`, `table.generateConsistencyToken()`, and `table.checkConsistency()` methods.
- Added new sample snippets.

## Fixes

- (#117): `setMetadata` partially updates, instead of overwriting previous entries.
- (#187, #195): Handle non-UTF8 Buffer filters.
- (#203): Stop converting row keys to UTF8 strings by default.
- (#207): Remove `row.data` property when a mutation is performed.
- Updated dependencies.

**Comment by stephenplusplus** [comment]
@JustinBeckwith CI error:

```
#!/bin/bash -eo pipefail
cd samples/
npm install
npm link ../
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @google-cloud/bigtable@0.15.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2018-06-28T20_50_42_341Z-debug.log
Exited with code 1
```

**Comment by stephenplusplus** [comment]
Maybe these two lines need to be reversed?:

```
npm install
npm link ../
```

Otherwise, the install step is having a crisis.

**Comment by JustinBeckwith** [comment]
It's failing because version `0.15.0` does not exist.  Flipping the order there would (probably) fix the issue :)

**Comment by stephenplusplus** [comment]
Tested locally before suggesting :)

**Comment by stephenplusplus** [comment]
@JustinBeckwith should I send a PR just for this repo, or is this one of those "ka-pow!"/PR parties, where all the repos get the change at the same time?

**Comment by JustinBeckwith** [comment]
Just this one for now is fine.  We can chat about the `ka-pow` one later 😆

**Comment by stephenplusplus** [comment]
Done in this PR! If we "Rebase and merge", we can preserve the individual commit that introduced it.

**Comment by stephenplusplus** [comment]
Unfortunately, I'm not getting anywhere :( Can you take a look? https://circleci.com/gh/googleapis/nodejs-bigtable/6848

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ jobs:
           name: Link the module being tested to the samples.
           command: |
             cd samples/
-            npm link ../
+            npm link @google-cloud/bigtable
```
Comment (JustinBeckwith) [inline .circleci/config.yml]: I am confused :) `npm link ../` will make sure `npm install` is run on the parent directory, AND do the linking.  Why switch to the named module?

Comment (stephenplusplus) [reply .circleci/config.yml]: Yeah, I thought so as well. The error persists with any configuration I've tried so far (including the one that was in the config.yml before I made any changes).

**Comment by sduskis** [comment]
Can we revert the samples upgrade until the deployment happens?

**Comment by sduskis** [comment]
@stephenplusplus: I added a magical `-g` to the samples install, and things workd.

**Comment by stephenplusplus** [comment]
Nice! If it works for @JustinBeckwith, it works for me!

**Comment by sduskis** [comment]
@JustinBeckwith or @stephenplusplus can we merge this?

**Pull request merged**